tests/components/channel_members_dropdown.test.jsx

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import ChannelMembersDropdown from 'components/channel_members_dropdown/channel_members_dropdown.jsx';
+
+describe('components/channel_invite_modal', () => {
```

Comment:
May be a typo error.  Should be: `components/channel_members_dropdown`

Reply:
Nice catch